src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -87,7 +88,18 @@ namespace OpenLoco::Vehicles::OrderManager
     Order* orders() { return reinterpret_cast<Order*>(getGameState().orders); }
     uint32_t& numOrders() { return getGameState().numOrders; }
 
-    void sub_470795(const uint32_t removeOrderTableOffset, const int16_t sizeOfRemovedOrderTable)
+    void shiftOrdersUp(const uint32_t offsetToShiftTowards, const int16_t sizeToShiftBy)
```

Comment:
lets call these left and right. I'm pretty sure both of these should be left shifts. The second argument is what will be the beginning after the rotate. So for both delete and free we want the second argument to be the offset + size which is shiftOrdersUp.

shiftOrdersDown looks wrong. It doesn't make sense you would need to have the end range be > numOrders to make some space. I think it would be something like this. 
 `std::rotate(&orders()[offsetToShiftFrom], &orders()[numOrders()], &orders()[numOrders() + sizeToShiftBy]);`

Reply:
Thanks Duncan. I've used your suggestion. Currently, `shiftOrdersDown` is unused; I suppose we'll start using it when we implement the vehicleOrderInsert command. Might tackle that later today.